PProPanel/ext.js

```diff
@@ -222,11 +222,11 @@ function loadJSX() {
 	var extensionPath = csInterface.getSystemPath(SystemPath.EXTENSION);
 
 	// load general JSX script independent of appName
-	const extensionRootGeneral = extensionPath + '/jsx/';
+	var extensionRootGeneral = extensionPath + '/jsx/';
```

Comment:
@mBeierl That would be _awesome_. [`let` is a bit of an odd case](http://caniuse.com/#feat=let) as it would require enabling experimental JavaScript features on Chrome 27 (CEP 5.2-6.0) and `strict` mode for Chrome 41...

On a side note, I just [posted](https://forums.adobe.com/thread/2353067) a bit of an explanation about the TypeScript Sample to Adobe's forums. Perhaps that would be a good place to start to understand the "Why" behind the setup.